pages/test_analytics.md

```diff
@@ -1,3 +1,7 @@
+---
+toc: true
```

Comment:
Test Analytics is a landing page so this should be false.

```suggestion
toc: false
```

Reply:
Also, landing pages are an interesting one and I think a good candidate to start using frontmatter with.

Perhaps in a separate ticket we could have something like a `page_layout` attribute with values like `default|landing_page`. 

Landing pages are currently using the `landing_page.html.erb` view layout, which could possibly be refactored to just one file with conditionals in the template. We could also get rid of the `landing_pages.rb` model as well.